static/app/components/teamSelector.tsx

```diff
@@ -46,6 +48,20 @@ const unassignedOption = {
   disabled: false,
 };
 
+const CREATE_TEAM_VALUE = 'CREATE_TEAM_VALUE';
+const createOption = {
+  value: CREATE_TEAM_VALUE,
+  label: 'Create team',
```

Comment:
```suggestion
  label: t('Create a team'),
```

Reply:
Hmm, in the rest of the product we don't use `a`.
Should I still change it?

Reply:
Ah, I was basing it on the label of the button we're replacing - your call then 👍
Let's make sure we translate it, though.